wordpress/wp-content/mu-plugins/cds-base/classes/Modules/Notify/ListManagerUserProfile.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace CDS\Modules\Notify;
+
+use WP_User;
+
+class ListManagerUserProfile
+{
+
+    public function __construct()
+    {
+    }
+
+    public static function register()
+    {
+        $instance = new self();
+
+        add_action('show_user_profile', [$instance, 'displayListManagerMeta']);
```

Comment:
this one only shows up for superadmins, right? It seems like we don't really need it, since we can't turn this off if we're superadmins (it is just checked again if you reload the page a couple times).

Reply:
Updated